src/components/Navbar/index.js

```diff
@@ -40,25 +32,27 @@ class Navbar extends React.Component<NavbarProps, NavbarState> {
       updateInfluences,
       updateInfluencers,
     } = this.props
-    const { searchIsEnabled } = this.state
 
     const about = React.createElement('a', { onClick: toggleAbout }, 'About')
     const logo = React.createElement('div', { onClick: goHome },
       React.createElement('img', { src: 'static/images/logo.svg' }),
       React.createElement('h1', {}, 'THE SPHERE OF MY INFLUENCE'))
 
-    const nav = React.createElement('nav', { 
-      onClick: () => this.toggleSearch() 
-    }, logo, React.createElement('div', { className: 'right' }, about))
+    const nav = React.createElement('nav', {}, 
```

Comment:
this is just to remove the optional rendering of the Search component; it should render all the time.